deeplabcut/generate_training_dataset/frame_extraction.py

```diff
@@ -136,12 +137,18 @@ def extract_frames(config,mode='automatic',algo='kmeans',crop=False,userfeedback
                 if opencv:
                     cap=cv2.VideoCapture(video)
                     fps = cap.get(5) #https://docs.opencv.org/2.4/modules/highgui/doc/reading_and_writing_images_and_video.html#videocapture-get
+                    if fps == 0:
+                        warn('Skipping %s\nIt has corrupt metadata, appears to be 0 fps' % video)
+                        continue
                     nframes = int(cap.get(7))
                     duration=nframes*1./fps
                 else:
                     #Moviepy:
                     clip = VideoFileClip(video)
                     fps=clip.fps
+                    if fps == 0:
```

Comment:
I am using `==` because I am comparing two _numbers_ . If I had used `is`, the if test would have raised an error for floating point numbers because I am comparing against an integer.

This code will print True:
```
a = 1.0
print(a == 1)
```
This code will print False
```
a = 1.0
print(a is 1)
```

`Is` is used when we want to ensure that the two objects we compare are an instance of the same class, which is the case for Boolean. With that said, this code will print True, when it maybe shouldn't:
```
a = 1
print(a == True)
```
This on the other hand will print False:
 ```
a = 1
print(a is True)
```
However this will also print False:
```
a = np.array([True, True, True])
print(a is True)
```

Note that `is` is also faster than `==`, but that shouldn't matter. Just a fun-fact.